Detectors/ITSMFT/ITS/base/src/GeometryTGeo.cxx

```diff
@@ -314,11 +320,17 @@ TGeoHMatrix* GeometryTGeo::extractMatrixSensor(int index) const
   //  mat->Print();
   // Restore the modeler state.
   gGeoManager->PopPath();
+
+  // account for the difference between sensitive layer and physical sensor ticknesses
+  static TGeoTranslation tra(0.,0.5*(Segmentation::SensorThickness-Segmentation::SensLayerThickness),0.);
+
```

Comment:
Hi @bovulpes, this accounts in the ITS matrix creation for the difference in thicknesses of the physical sensor (where the charge deposit should be processed) and of the layer of effective charge collection (hit is assigned to the middle of this layer). I assume something similar should be done also for MFT matrices, to avoid ~12 \mum bias in hit position (along the beam in your case), but since I was not sure what local frame do you use in the end, I did not modify MFT. Could you check please?